_includes/footer2.html

```diff
@@ -0,0 +1,87 @@
+<footer class="footer">
```

Comment:
can you use a better name than footer2 please?
add an inline code comment for why a second one is needed

Reply:
i dont want loose old navigation because in design is different so i created footer2 and nav-bar 2,its just temporarily , i will deleted it